inspire_mitmproxy/http.py

```diff
@@ -178,6 +178,9 @@ def __repr__(self):
         return f'MITMRequest("{self.url}", "{self.method}", ' \
             f'headers={repr(self.headers)}, body="{self.body}")'
 
+    def __getitem__(self, item):
+        return self.to_dict()[item]
```

Comment:
s/uri/url/ in the cassettes and use getattr here instead (to avoid getting a copy of the values back, instead of the reference to them).